.github/workflows/prchecks.yml

```diff
@@ -46,9 +45,6 @@ jobs:
         with:
           upload-snapshots: true
 
-      # - name: Document
-      #   run: Rscript -e 'roxygen2::roxygenise()'
-
```

Comment:
Is this not needed anymore?

Reply:
This code was already commented out in the script, so I removed it. Please let me know if you would like me to bring it back. This job is run in the prcommands actions file.